agent-deploy/Dockerfile

```diff
@@ -135,6 +139,15 @@ RUN curl -LO https://go.dev/dl/go${GOLANG_VERSION}.linux-amd64.tar.gz && \
     rm -f go${GOLANG_VERSION}.linux-amd64.tar.gz
 ENV PATH="$PATH:/usr/local/go/bin"
 
+# Install vault: https://github.com/hashicorp/vault/blob/main/CHANGELOG.md https://releases.hashicorp.com/vault
+RUN set -x \
```

Comment:
Even though this looks harmless in this context, I fear a script/scanner will eventually complain one day or the other about `set -x`
Since it doesn't serve much purpose anyway, could you remove it?